Clinical-Genomics/scout
# Fix #1395. Inactivate rerun cases only on re-upload.

This PR fixes a bug.

**How to test**:
1. Activate (or archive) a case.
1. Request rerun, and ensure that case does not become inactive.
1. See that the case gets a rerun-pending badge on cases list.
1. Re-upload case, e.g.  ```scout --demo load case --update scout/demo/643594.config.yaml```
and ensure that it now becomes inactive, a rerun badge is shown but no pending badge.

**Expected outcome**:
The functionality should be working.

**Review:**
